content/en/agent/proxy.md

```diff
@@ -355,11 +357,11 @@ frontend logs_http_frontend
     default_backend datadog-logs-http
 
 # If sending logs with use_tcp: true
-frontend logs_frontend
-    bind *:10514
-    mode tcp
-    option tcplog
-    default_backend datadog-logs
+# frontend logs_frontend
+#    bind *:10514
+#    mode tcp
+#    option tcplog
+#    default_backend datadog-logs
```

Comment:
Do we still need this? 

As this page is made for HTTPS now so if i'm correct this is still the TCP section

Reply:
yes that's why I commented them, but I should just remove it.

Reply:
removed